open-metadata-resources/open-metadata-labs/lineage/lineage.ipynb

```diff
@@ -0,0 +1,692 @@
+    "configureDescriptiveName(adminPlatformURL, adminUserId, olsServerName, metadataCollectionName)\n",
+    "configureCohortMembership(adminPlatformURL, adminUserId, olsServerName, cocoCohort)\n",
+    "\n",
+    "print(\"\\nDone.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "139224c2-44a2-4f36-94e0-6d8b7260341b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO: consider placing the below configuration into function calls like the other configuration,\n",
+    "# once functions exist to do the open lineage configuration...\n",
+    "\n",
+    "olsConfigDocument = \"\"\"\n",
+    "{\n",
+    "    \"class\": \"OpenLineageConfig\",\n",
+    "    \"openLineageId\": 0,\n",
+    "    \"openLineageDescription\": \"Open Lineage Service is used for the storage and querying of lineage\",\n",
+    "    \"inTopicName\": \"server.cocoMDS4.omas.assetlineage.outTopic\",\n",
+    "    \"inTopicConnection\": {\n",
+    "        \"class\": \"Connection\",\n",
+    "        \"headerVersion\": 0,\n",
+    "        \"connectorType\": {\n",
+    "            \"class\": \"ConnectorType\",\n",
+    "            \"headerVersion\": 0,\n",
+    "            \"type\": {\n",
+    "                \"class\": \"ElementType\",\n",
+    "                \"headerVersion\": 0,\n",
+    "                \"elementOrigin\": \"LOCAL_COHORT\",\n",
+    "                \"elementVersion\": 0,\n",
+    "                \"elementTypeId\": \"954421eb-33a6-462d-a8ca-b5709a1bd0d4\",\n",
+    "                \"elementTypeName\": \"ConnectorType\",\n",
+    "                \"elementTypeVersion\": 1,\n",
+    "                \"elementTypeDescription\": \"A set of properties describing a type of connector.\"\n",
+    "            },\n",
+    "            \"guid\": \"3851e8d0-e343-400c-82cb-3918fed81da6\",\n",
+    "            \"qualifiedName\": \"Kafka Open Metadata Topic Connector\",\n",
+    "            \"displayName\": \"Kafka Open Metadata Topic Connector\",\n",
+    "            \"description\": \"Kafka Open Metadata Topic Connector supports string based events over an Apache Kafka event bus.\",\n",
+    "            \"connectorProviderClassName\": \"org.odpi.openmetadata.adapters.eventbus.topic.kafka.KafkaOpenMetadataTopicProvider\",\n",
+    "            \"recognizedConfigurationProperties\": [\n",
+    "                \"producer\",\n",
+    "                \"consumer\",\n",
+    "                \"local.server.id\",\n",
+    "                \"sleepTime\"\n",
+    "            ]\n",
+    "        },\n",
+    "        \"endpoint\": {\n",
+    "            \"class\": \"Endpoint\",\n",
+    "            \"headerVersion\": 0,\n",
+    "            \"address\": \"server.cocoMDS4.omas.assetlineage.outTopic\"\n",
+    "        },\n",
+    "        \"configurationProperties\": {\n",
+    "            \"producer\": {\n",
+    "                \"bootstrap.servers\": \"lab-kafka:9092\"\n",
```

Comment:
this works in k8s but will fail when run locally or in docker compose. See the config we use for the Event Bus

So we probably want something like/:
```
    "        \"bootstrap.servers\": eventBusURLroot\n",
```

Also repeated for the consumer beneath